src/kwlib.sh

```diff
@@ -1,5 +1,9 @@
 # NOTE: src/kw_config_loader.sh must be included before this file
 
+# Array with compression programs accepted by tar
+declare -ga compression_programs=('gzip' 'bzip2' 'lzip' 'lzma' 'lzop' 'zstd' 'xz'
+  'auto-compress')
```

Comment:
Can you confirm if we need to add a dependency in the `setup.sh` for some of these compression options? For example, I think lzma requires something extra.

Reply:
From what I searched, lzip and zstd don't come by default on Ubuntu, so users need to install them before using them with tar